src/WebUI/Common/KubeSummary.tsx

```diff
@@ -54,11 +54,18 @@ export interface IKubernetesContainerState {
     svcFilter: Filter;
 }
 
+export interface ImageLocations {
```

Comment:
i am thinking the following way, your approach is also fine, it is just another thought.

let us have enum, and a function.
now i call the function with enum, if the output is notnull then use that otherwise use default.
ofcourse for future, adding a new image, have to be changed in the enum, and in the consumer we have to use that enum. advantage of this approach is, caller can always have a default switch of the enum and can send a default image. in this case the devops should always know the name of the properyu